### ISSUE TITLE
Issue with feed

### ISSUE THREAD

**Issue by DannyThomas1** [adrianhajdin/project_next_13_ai_prompt_sharing#22]
Does anyone have an issue where the feed is not pulling all the prompts from the db after deploying it to Vercel? If I click on a profile there are all the prompts from that user but on the feed page, it's missing some that were created by that user. It only happens on the Vercel deployed site, on my local environment there's no issue with the feed

**Comment by marinkres** [comment]
Yeah, I have the same issue. Don't know how to fix it, I'm guessing it has something to do with cacheing.

**Comment by krishan1007** [comment]
same issue

**Comment by Sanket2055** [comment]
yess

**Comment by nalvadev** [comment]
same here

**Comment by NikChhillar** [comment]
Same issue, if anyone find any solution to it, please share

**Comment by Sanket2055** [comment]
@adrianhajdin

**Comment by uns4inted** [comment]
Same issue. I've created pull request with fix and explanation for this problem, you can make changes in your project according to changes:
https://github.com/adrianhajdin/project_next_13_ai_prompt_sharing/pull/21

**Comment by DannyThomas1** [comment]
Ya i ran into that issue when i was trying to build the app (I built it using TS) and saw your PR so thanks for that fix! But it didn't solve my feed issue still unfortunately 😞

**Comment by marinkres** [comment]
Yeah sadly didn't fix it for me either

**Comment by uns4inted** [comment]
> Ya i ran into that issue when i was trying to build the app (I built it using TS) and saw your PR so thanks for that fix! But it didn't solve my feed issue still unfortunately 😞

Thanks for your reply, look's like there is something else I did before in my code that make this fix work for me. 😅
I will update my PR and answer here when I figure this out.

**Comment by Sanket2055** [comment]
i am getting the following errors in building . 
```

Module not found: Can't resolve 'supports-color' in '/vercel/path0/node_modules/debug/src'
Module not found: Can't resolve 'kerberos' in '/vercel/path0/node_modules/mongoose/node_modules/mongodb/lib'
Module not found: Can't resolve '@mongodb-js/zstd' in '/vercel/path0/node_modules/mongoose/node_modules/mongodb/lib'
Module not found: Can't resolve '@aws-sdk/credential-providers' in '/vercel/path0/node_modules/mongoose/node_modules/mongodb/lib'
Module not found: Can't resolve 'snappy' in '/vercel/path0/node_modules/mongoose/node_modules/mongodb/lib'
Module not found: Can't resolve 'aws4' in '/vercel/path0/node_modules/mongoose/node_modules/mongodb/lib'
Module not found: Can't resolve 'mongodb-client-encryption' in '/vercel/path0/node_modules/mongoose/node_modules/mongodb/lib'

```
anyone else getting the same error??

**Comment by uns4inted** [comment]
Found the workaround for the problem with cahing prompts data for the Feed page. Check the PR: https://github.com/adrianhajdin/project_next_13_ai_prompt_sharing/pull/24 for the solution.

This one was not obvious, so all info I get from Next.js docs and researching about caching is simply don't fix the issue. Maybe it's some kind of bug in Next, but I'm really don't get the root of the problem.
So the only way to make things work for me - is this workaround.

**Comment by Marium-dotcom** [comment]
I guess the issue is with Vercel itself because I deployed the app on Netlify and the feed got updated. But It didn't let me to edit or search.

**Comment by Tanimou** [comment]
> Found the workaround for the problem with cahing prompts data for the Feed page. Check the PR: #24 for the solution.
> 
> This one was not obvious, so all info I get from Next.js docs and researching about caching is simply don't fix the issue. Maybe it's some kind of bug in Next, but I'm really don't get the root of the problem. So the only way to make things work for me - is this workaround.

Thanks a lot. it worked for me too. been like 2 weeks being stucked on this bug

**Comment by jeeknowme** [comment]
For anybody else that also haven't made it worked
 const response = await fetch("/api/prompt", {
        next: { revalidate: 10, cache: "no-store" },
      });
      
      this is in the feed.jsx

**Comment by zzh315** [comment]
> 

 did it work for you?

**Comment by zzh315** [comment]
> For anybody else that also haven't made it worked, this is in the feed.jsx
> 
> const response = await fetch("/api/prompt", { next: { revalidate: 10, cache: "no-store" }, });

didn't work  for me.did it work for you ?

**Comment by jeeknowme** [comment]
Yes it did
